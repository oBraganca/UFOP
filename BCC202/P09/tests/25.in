25
DaviLucca 174443 4.6
Emanuel 315832 1.1
Sarah 255751 6.4
Antonia 564706 4.7
Livia 169702 3.5
Julia 728184 7.3
Isabelly 468437 7.8
Henrique 64034 3.7
Beatriz 579384 0.8
Tomas 312603 7.5
Enzo 583042 2.9
Lucca 695718 9.4
Cecilia 746435 6.8
Felipe 898457 4.9
Mariah 824944 3.3
Eduarda 509422 9.6
Fernanda 149760 2.5
MariaVitoria 580385 3.2
Sophia 134106 1.2
Raul 843260 4.9
Leonardo 766248 1.6
Luna 759302 10.0
Henry 928688 8.3
Elisa 418900 5.0
Heloise 191055 6.4
