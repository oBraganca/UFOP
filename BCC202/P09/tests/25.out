Beatriz 579384 0.8
Emanuel 315832 1.1
Sophia 134106 1.2
Leonardo 766248 1.6
Fernanda 149760 2.5
Enzo 583042 2.9
MariaVitoria 580385 3.2
Mariah 824944 3.3
Livia 169702 3.5
Henrique 64034 3.7
DaviLucca 174443 4.6
Antonia 564706 4.7
Raul 843260 4.9
Felipe 898457 4.9
Elisa 418900 5.0
Heloise 191055 6.4
Sarah 255751 6.4
Cecilia 746435 6.8
Julia 728184 7.3
Tomas 312603 7.5
Isabelly 468437 7.8
Henry 928688 8.3
Lucca 695718 9.4
Eduarda 509422 9.6
Luna 759302 10.0
